libipl/p10/ipl0.C

```diff
@@ -377,19 +385,84 @@ static int ipl_alignment_check(void)
 static int ipl_set_ref_clock(void)
 {
 	struct pdbg_target *proc;
+    struct pdbg_target *clock_target;
+    enum pdbg_target_status status;
 	int rc = 0;
+    uint8_t clock_count = 0;
 
 	if (ipl_type() == IPL_TYPE_MPIPL)
 		return -1;
 
 	ipl_log(IPL_INFO, "Istep: set_ref_clock: started\n");
 
+	ipl_log(IPL_INFO, "Istep: soft reset clock, and verify clock status register\n");
+	pdbg_for_each_class_target("oscrefclk", clock_target) {
+        uint8_t data;
```

Comment:
This new function need to run in reboot path?

Reply:
yes